backend/src/main/java/ch/puzzle/okr/repository/ObjectiveRepository.java

```diff
@@ -4,13 +4,12 @@
 import ch.puzzle.okr.models.Quarter;
 import ch.puzzle.okr.models.Team;
 import java.util.List;
-import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
-public interface ObjectiveRepository extends CrudRepository<Objective, Long> {
+public interface ObjectiveRepository extends DeleteRepository<Objective, Long> {
 
-    Integer countByTeamAndQuarter(Team team, Quarter quarter);
+    Integer countByTeamAndQuarterAndIsDeletedFalse(Team team, Quarter quarter);
```

Comment:
is only used for testing, remove and remove tests or make separate task, up to you